src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/PhysicalMarker/MarkerDetectorCoordinateService.cs

```diff
@@ -183,20 +183,30 @@ private async Task OnDiscoverCoordinatesImplAsync(CancellationToken cancellation
             }
         }
 
+#if UNITY_EDITOR
         private async Task OnDiscoverCoordinatesEditorAsync(CancellationToken cancellationToken, int[] idsToLocate = null)
         {
-            if (idsToLocate == null ||
-                idsToLocate.Length != 1)
+            if (UnityCompositorInterface.IsArUcoMarkerDetectorSupported())
             {
-                DebugLog("Running the MarkerDetectorCoordinateService in the editor only supports one coordinate id");
+                await OnDiscoverCoordinatesImplAsync(cancellationToken, idsToLocate);
                 return;
```

Comment:
this UnityCompositorInterface call may need to be in a try catch if folks try and run mobile + hololens simulation in the editor.